r4/coverage-requirements-discovery/Config.toml

```diff
@@ -0,0 +1,40 @@
+# CDS definition with one prefetch template
+[[cds_services]]
```

Comment:
readme says we need to have fully qualified package name here. Shall we make it consistent?